package.js

```diff
@@ -3,13 +3,20 @@ Package.describe({
   name: "practicalmeteor:mocha",
   summary: "Write package tests with mocha and run them in the browser or from the command line with spacejam.",
   git: "https://github.com/practicalmeteor/meteor-mocha.git",
-  version: '2.1.0_7'
+  version: '2.1.1-rc.1',
+  testOnly: true
+});
+
+
+Npm.depends({
```

Comment:
I think perhaps I added that line in error.

However it *is* possible to use Mocha in this way, for instance this package does it: https://github.com/DispatchMe/meteor-mocha-phantomjs